jans-lock/cedarling/tests/get-str.rs

```diff
@@ -1,10 +1,9 @@
-use cedarling::utils;
-use wasm_bindgen::UnwrapThrowExt;
+use cedarling::http::{self, ResponseEx};
 use wasm_bindgen_test::*;
 
 #[wasm_bindgen_test]
 async fn test() {
-	let data = utils::get("https://site-scraper.sokorototo.workers.dev/").await.unwrap_throw();
-	let string = std::str::from_utf8(&data).unwrap_throw();
+	let data = http::get("https://site-scraper.sokorototo.workers.dev/", &[]).await.unwrap();
```

Comment:
Needs to be done via mocking responses, not real requests.

Reply:
It's a personal service, does it still need to use mocking?

Reply:
Mm yeah, it will still be using traffic that might cost depending on where we run tests.

Reply:
I don't think it will hit the service 1M times a month. Then it would cost me 5$